patches/chrome-browser-updates-announcement_notification-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/updates/announcement_notification/BUILD.gn b/chrome/browser/updates/announcement_notification/BUILD.gn
+index fe6f065a756ddfc0bd651ec4387948a368d09867..3193cdf414407b508ae6991ad4159fb3d141d686 100644
+--- a/chrome/browser/updates/announcement_notification/BUILD.gn
++++ b/chrome/browser/updates/announcement_notification/BUILD.gn
+@@ -20,6 +20,7 @@ source_set("announcement_notification") {
+ 
+   deps = [
+     "//base",
++    "//chrome/app:generated_resources",
```

Comment:
Yes, this looks like upstream bug w/o this for android build.
I think their buildbot didn't see this error by luck? :)